bitnami/cassandra/templates/service.yaml

```diff
@@ -27,7 +27,7 @@ spec:
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
   {{- end }}
   {{- if and (eq .Values.service.type "LoadBalancer") (not (empty .Values.service.loadBalancerSourceRanges)) }}
-  loadBalancerSourceRanges: {{ .Values.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges: {{- toYaml $.Values.service.loadBalancerSourceRanges | nindent 4 }}
```

Comment:
There is no need to use `$.Values` here as this is not inside a `range` loop, could you please replace it with `.Values`?

Reply:
Have replaced `$.Values` from `.Values`. Thank You